prime-router/src/main/resources/metadata/hl7_mapping/OML_O21/base/order/observation-request/observation/observation.yml

```diff
@@ -0,0 +1,21 @@
+# $schema: ./../../../../../../../../../metadata/json_schema/fhir/fhir-to-hl7-mapping.json
+
+constants:
+  observationPath: '%resource.supportingInfo.resolve().ofType(Observation).where(subject.resolve().id = %resource.subject.resolve().id)'
+
+elements:
+
+  - name: observation-result
+    resource: '%observationPath'
+    resourceIndex: resultIndex
+    required: false
+    schema: classpath:/metadata/hl7_mapping/resources/Observation/OBX.yml
+    constants:
+      hl7OBXField: '/ORDER(%{orderIndex})/OBSERVATION_REQUEST/OBSERVATION(%{resultIndex})/OBX'
+      hl7ObservationPath: '/ORDER(%{orderIndex})/OBSERVATION_REQUEST/OBSERVATION(%{resultIndex})'
+
+  - name: observation-participation-information
+    resource: '%observationPath.device.resolve().where(udiCarrier.exists())' # correct?
```

Comment:
After a second look I don't see any reason to keep the `.where(udiCarrier.exists()` clause in this. Originally I put that in because of how ORU was mapping Observation.device.

Reply:
After a third look, this is something I will be taking care of in the follow up ticket if feasible. 

What's going on: HL7 OML Observation data can be mapped to FHIR Device in two different ways. OBX.18 maps to a device resource which is present in the integration test data. OML Observations also can be partnered with a PRT segment, which would map to its own FHIR Device. However, in the later scenario this mapping only happens if PRT.10 is present, which maps to Device.udiCarrier. 
Because there is no PRT data in this first pass at the OML integration test, it makes more sense to take care of this in the second ticket.

Commented out as a todo: https://github.com/CDCgov/prime-reportstream/pull/15910/commits/a57270e789fc8aee206bd18990cf36c0ac9e6072